Day1/index.js

```diff
@@ -0,0 +1,40 @@
+const {sum,subtract,multiply,divide} = require('./lib/math');
+
+
+
+const fs = require("fs");
+
+
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+rl.question("Enter the first number: ", (firstNumber) => {
+  rl.question("Enter the second number: ", (secondNumber) => {
+    const parsedFirst = parseFloat(firstNumber);
+    const parsedSecond = parseFloat(secondNumber);
+    console.log(`You entered ${parsedFirst} and ${parsedSecond}. And result of add , subtract, multiply, divide operrations are following:`);
+   console.log(sum([parsedFirst, parsedSecond]));
+   console.log(subtract(parsedFirst, parsedSecond));
+   console.log(multiply(parsedFirst, parsedSecond));
+   console.log(divide(parsedFirst, parsedSecond).toFixed(2));
+   const data = `
+Opeartion , Result,
+add ${sum([parsedFirst, parsedSecond])}
```

Comment:
@anantmishra-successive You are doing all the operations twice.Follow DRY principle.

Reply:
![inputline](https://github.com/anantmishra-successive/Backend_Training/assets/162098832/2f44e6b2-cdd8-4f0f-9510-c0c8b7713478)
Ok sir